business_requirement_deliverable_project/models/project.py

```diff
@@ -26,11 +26,16 @@ def generate_project_wizard(self):
                     config settings"""))
         lines = []
         for br in br_ids:
-            if br.state not in ['approved', 'cancel', 'done']:
+            if br.state not in [
+                    'approved',
+                    'stakeholder_approval',
+                    'cancel',
+                    'done'
+            ]:
                 raise ValidationError(
                     _("""All business requirements of the project should
                         be approved/canceled/done"""))
-            if br.state != 'approved':
+            if br.state not in ['approved', 'stakeholder_approval']:
```

Comment:
Shouldnot it  only be "Stakeholder approval"?
In principle it should be possible to generate project when only internally approved but this is dangerous somehow